src/control/logging/logging_test.go

```diff
@@ -25,12 +25,59 @@ package logging_test
 import (
 	"bytes"
 	"regexp"
+	"strings"
 	"testing"
 
 	"github.com/daos-stack/daos/src/control/logging"
 )
 
-func TestStandardFormat(t *testing.T) {
+type testWrap struct {
+	log logging.Logger
+}
+
+func (tw *testWrap) Debug(msg string) {
+	tw.log.Debug(msg)
+}
+
+func (tw *testWrap) Debugf(fmtStr string, args ...interface{}) {
+	tw.log.Debugf(fmtStr, args...)
+}
+
+// NB: Keep this test at the top of the file to avoid having to
+// update the line numbers all the time.
```

Comment:
Not sure if it's feasible, but maybe a simple set of regular expressions could be used to remove this fragility

Reply:
Well, I wanted to be sure that the reported line number matched the actual line number. Couldn't think of a more robust way to do that. Open to suggestions, though.

Reply:
Could get the line number with runtime.Caller on the line before or after the call being tested, then add/subtract one, perhaps? I just don't like the idea of this breaking when we have to add a new import or something.

Reply:
Yeah, good call. I've updated the test to be insulated from line number changes.